tservice/test_utils/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "test_utils"
+# Version is kept here in this local only crate to make it easy to automated
+# updating the version string in all the crates.
+version = "0.2.0-pre0"
+edition = "2021"
+authors = ["MobileCoin"]
+rust-version = "1.62.1"
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/mobilecoinfoundation/sgx"
+description = "Test utilities for the rust wrappers of the `sgx_tservice` library."
+
+categories = ["development-tools::testing"]
+keywords = ["sgx", "test"]
```

Comment:
I'm ambivalent about this... If we're going to go down the "grouping" route I'd like to 1. make more groups and 2. make everything look the same.

Reply:
resolving for now as this is in other cargo.toml files and should have a dedicated normalization effort